Dockerfile

```diff
@@ -28,6 +28,10 @@ RUN apt-get update -qq && \
 # http://unix.stackexchange.com/questions/195975/cannot-force-remove-directory-in-docker-build
 #        && rm -rf /var/lib/apt/lists
 
+# adding custom locales to provide backward support with scrapy cloud 1.0
+RUN echo 'en_US.UTF-8 UTF-8\nde_DE.UTF-8 UTF-8\nnl_NL.UTF-8 UTF-8\nfr_FR.UTF-8 UTF-8' \
+    >> /etc/locale.gen && locale-gen
```

Comment:
I think it was picked the other base image because it's official python2 image.

Don't like it because the more distributions we have - the more time consuming is support. There could be some bugs (in Scrapy, any other python or deb package) that happen in one distribution and don't happen in the other. Configuration is slightly different. Package managers could be slightly different. Right now we have 2 distributions - Ubuntu and Debian here. Some time ago we discussed an option to use Alpine as a base image - +1 distribution. Don't you agree that dealing with 1 distribution in stacks would have been more straightforward?